roles/prereqs/libsemanagepython/meta/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+
+dependencies:
+  - role: prereqs/packages
```

Comment:
this doesn't actually exist anymore (there is no prereqs/packages role in the current commit, which may be my fault).  also, why is libsemanagepython using this method, but libselinuxpython is using a task?  The two roles should do approximately the same thing.  So, either put back the prereqs/packages role (and modify libselinuxpython to use it), or make libsemanagepython install the package itself...